packages/map-template/src/components/WebComponentWrappers/RouteInstructionsStep/RouteInstructionsStep.jsx

```diff
@@ -87,8 +93,10 @@ const RouteInstructionsStep = forwardRef(({ totalSteps, activeStep, previous, or
         }
     }, [substepsOpen]);
 
+
     return <mi-route-instructions-step
         ref={elementRef}
+        show-toggle-button={kioskLocation === undefined || (kioskLocation !== undefined && !isDesktop)}
```

Comment:
Could we also use the new custom hook here? 🤔

Reply:
Yess! ✅